dijet/selection/stats.py

```diff
@@ -38,10 +36,10 @@ def dijet_increment_stats(
     }
 
     if self.dataset_inst.is_mc:
-        weight_map["sum_mc_weight"] = events.mc_weight # weights of all events
-        weight_map["sum_mc_weight_selected"] = (events.mc_weight, event_mask) # weights of selected events
-        
-    # baut alles auf weight_map zu weight_map_per_process    
+        weight_map["sum_mc_weight"] = events.mc_weight  # weights of all events
+        weight_map["sum_mc_weight_selected"] = (events.mc_weight, event_mask)  # weights of selected events
+
+    # baut alles auf weight_map zu weight_map_per_process
```

Comment:
Huch, ein Kommentar auf Deutsch :) Let's stick to English though for accessibility and consistency reasons.

Reply:
Replaced with an englisch comment